modules/admin-ui/src/main/java/org/opencastproject/adminui/endpoint/SeriesEndpoint.java

```diff
@@ -941,7 +941,8 @@ public Response deleteSeriesTheme(@PathParam("seriesId") String seriesID) throws
           @RestResponse(responseCode = SC_OK, description = "The ACL has been successfully applied"),
           @RestResponse(responseCode = SC_BAD_REQUEST, description = "Unable to parse the given ACL"),
           @RestResponse(responseCode = SC_NOT_FOUND, description = "The series has not been found"),
-          @RestResponse(responseCode = SC_INTERNAL_SERVER_ERROR, description = "Internal error") })
+          @RestResponse(responseCode = SC_INTERNAL_SERVER_ERROR, description = "Internal error"),
+          @RestResponse(responseCode = SC_UNAUTHORIZED, description = "If the current user is not authorized to perform this action") })
```

Comment:
Maybe you mean The current user is not authorized. Not a question if he is authorized?

Reply:
I think I just copied this from somewhere else, but yeah, the description could be better. Gonna fix this.